app/model/editions/templates/AustralianEdition.scala

```diff
@@ -56,14 +56,15 @@ object AustralianEdition {
       // Crosswords
       FrontCrosswords -> Daily(),
     ),
-    zoneId = ZoneId.of("Europe/London"),
-    availability = Daily(),
     capiQueryPrefillParams = CapiQueryPrefillParams(
       timeWindowConfig = TimeWindowConfigInDays(
         startOffset = 0,
         endOffset = 0
       )
-    )
+    ),
+    zoneId = ZoneId.of("Europe/London"),
+    availability = Daily(),
+    7
   )
```

Comment:
it's not clear for me what 7 means in that case
i like syntax name = ...
so the confis are easy to read